tensorflow/python/keras/layers/pooling.py

```diff
@@ -370,6 +370,13 @@ class MaxPooling2D(Pooling2D):
             [[10.],
              [11.],
              [12.]]]], dtype=float32)>
+             
+  Usage Example:
+  
+  >>> model = tf.keras.models.Sequential()
+  >>> model.add(tf.keras.layers.Conv2D(32, kernel_size=(3, 3), 
+  ...    input_shape=(28,28,1)))
+  >>> model.add(tf.keras.layers.MaxPooling2D(pool_size=(2, 2)))
```

Comment:
Shouldn't this print something?

Usually for a doctest you want to have some code and some output so when the test runs it checks that the output during the test is the same as the one that is on the file.

Probably create a toy input before creating the model and then run `model.fit` on the toy input and print the output tensor

Reply:
HI @mihaimaruseac, thanks for the really quick reply. I have made some changes so that an output is given but I'm not sure that I have done the correct thing here. Should I be making a prediction from the trained model after the code I added?

Reply:
Yes, please.

Also, instead of having the image created from random, please make it a small constant using `tf.constant`. This way, when the random test seed changes the test can still pass.

Reply:
Hi, thanks for the fast reply. How should I be making predictions as model.predict returns a numpy array instead of a tensor?

Reply:
You can print the numpy array too.

Reply:
I used tf.constant for the input image, but the code still outputs different array each time I run it. I have added seeds and turned of shuffle and it still outputs different each time. Do you know what I need to do to fix this? This is what I have at the moment:

>>> input_image = tf.constant([[[[1.], [1.], [2.], [4.], [2.], [4.], [2.]],
...                            [[2.], [2.], [3.], [2.], [2.], [1.], [2.]],
...                            [[4.], [1.], [1.], [1.], [1.], [2.], [2.]],
...                            [[2.], [2.], [1.], [4.], [2.], [3.], [4.]],
...                            [[1.], [4.], [1.], [1.], [2.], [3.], [2.]],
...                            [[1.], [4.], [2.], [3.], [1.], [2.], [3.]],
...                            [[3.], [4.], [1.], [2.], [3.], [1.], [4.]]]]) 
>>> output = tf.constant([[[[1], [0]],
...                       [[0], [1]]]]) 
>>> model = tf.keras.models.Sequential()
>>> model.add(tf.keras.layers.Conv2D(1, kernel_size=(3, 3), 
...    input_shape=(7,7,1)))
>>> model.add(tf.keras.layers.MaxPooling2D(pool_size=(2, 2)))
>>> model.compile('adam', 'mean_squared_error')
>>> model.fit(input_image, output, steps_per_epoch=1, 
...    shuffle=False)
>>> model.predict(input_image, steps=1)
array([[[[1.7140688 ], [1.3748764 ]],
        [[0.08504223], [0.8180907 ]]]], dtype=float32)

Thanks for the help

Reply:
@mihaimaruseac I also have another PR open at the moment (#35715 ) in which some of the changes here may apply. Could you advise whether they do or of it is different for that one?